src/main/resources/application.yaml

```diff
@@ -117,7 +117,7 @@ springdoc:
 logging:
   level:
     uk.gov.hmcts.reform.authorisation.filters: DEBUG
-
+    root: DEBUG
```

Comment:
I don't think root level should be set to DEBUG. Instead should be set to INFO. If set to DEBUG, logs will be filled with too much of details. We tend to enable/set this to DEBUG only when troubleshooting an issue.